Documentation/Blazorise.Docs/Pages/Docs/Extensions/TreeView/TreeViewPage.razor

```diff
@@ -192,6 +219,9 @@
     <DocsAttributesItem Name="AutoExpandAll" Type="bool" Default="false">
         Defines if the treenode should be automatically expanded. Note that it can happen only once when the tree is first loaded.
     </DocsAttributesItem>
+    <DocsAttributesItem Name="Virtualize" Type="bool" Default="false">
+        Defines if hidden treenodes should be visible. See <Anchor To="https://learn.microsoft.com/en-us/aspnet/core/blazor/components/virtualization">docs for Virtualization</Anchor>.
```

Comment:
> Defines if hidden treenodes should be visible. 

Are we sure this conveys the exact meaning of what `Virtualize` does? I'd say it's something along the lines of rendering the nodes as you scroll through the data limiting the number of html elements that are rendered at a time which in turns helps page performance when handling huge sources of data?